components/NavigationBar/tests/unit/NavigationBarButtonTitleFontTests.swift

```diff
@@ -26,6 +26,14 @@ class NavigationBarButtonTitleFontTests: XCTestCase {
     navigationBar = MDCNavigationBar()
   }
 
+  private func recursiveSubviews(of superview: UIView) -> [UIView] {
```

Comment:
Unit-test for this to make sure we don't have infinite loops or something silly?